compiler-plugin/src/main/resources/rules.json

```diff
@@ -0,0 +1,12 @@
+[
+  {
+    "id": 1,
+    "kind": "VULNERABILITY",
+    "description": "Avoid allowing default resource accessor"
```

Comment:
I would like to have a descriptive reason and resolution for these issues. Even though some issues are straight forward, we need to at least mention the reason for marking this as an vulnerability and how we can resolve this issue. Can you check the possibility of configuring them as well?  

For example if you check this Java sonar report: https://sonarcloud.io/project/issues?sinceLeakPeriod=true&issueStatuses=OPEN%2CCONFIRMED&pullRequest=2202&id=ballerina-platform_module-ballerina-http&open=AZLRrEMyf0DC8seLHPIr&tab=code
There is a section called `Why`.

Reply:
+1
@MaryamZi, @SasinduDilshara and I had a chat on this few days back. I believe Scan tool team already have plan to do this.

Reply:
Do we have an issue for this?

Reply:
There is an issue for report improvement. I have added this as a comment to track: https://github.com/ballerina-platform/static-code-analysis-tool/issues/17#issuecomment-2458920611

Reply:
Ack 👍🏼